dashmips/__main__.py

```diff
@@ -61,8 +61,8 @@ def main_docs(args: argparse.Namespace) -> int:
     if print_both or args.syscalls:
         # Syscall printer
         print("Syscalls")
-        print(f"{'name':15}{'number':<10}{'description'}")
-        print(f"{'----':15}{'------':<10}{'-----------'}")
+        print(f"{'name':15}{'$v0':<10}{'description':<30}{'arguments/result'}")
+        print(f"{'----':15}{'---':<10}{'-----------':<30}{'----------------'}")
```

Comment:
just take a crack at this and dw if it becomes too involved:
Can we print this like a markdown table? then we can just copy pasta to the docs to put up online in the wiki etc.
```
| name         | $v0 | arguments/results                                                         |
|--------------|-----|---------------------------------------------------------------------------|
| print_string | 08  | $a0 = address of input buffer $a1 = maximum number of characters to read. |
| ...          | ..  | ...                                                                       |
```

Reply:
![Syscall Table](https://user-images.githubusercontent.com/44068487/88086157-c19ca900-cb3b-11ea-8bf4-2b9e963ba7d2.png)